shell/probe/raindrop.ps1

```diff
@@ -40,22 +40,25 @@ while ($true) {
         }
         $Response = Invoke-WebRequest -URI $Api -UseBasicParsing -Headers $Headers -MaximumRedirection 1 -OutFile $Vst -PassThru
         $Test = $Response.BaseResponse.ResponseUri.AbsolutePath.Split("/")[-1].Split("_")[0]
-    
+        $Dat = ""
```

Comment:
I don't think we need to set Dat to empty before the loop and on each loop. 
removing both ``$Dat=""`` lines should post an empty/null to the API, right? if that's the case, all good, as the API handles that (and that matches nocturnal's behavior).

Reply:
no, PowerShell throws a "Object reference not set to an instance of an object" error on `Invoke-WebRequest` if you execute it with Null values in the `$Headers`. We have to set it to a blank string